src/main/java/lotto/domain/LottoNumbers.java

```diff
@@ -44,7 +45,7 @@ public boolean hasLottoNumber(LottoNumber lottoNumber) {
                 .contains(lottoNumber.getValue());
```

Comment:
getter를 사용하지 않고 다음과 같이 변경해도 괜찮을 것 같아요 :)
```suggestion
                return value.contains(lottoNumber);
```